분석 UI/UX 수정

lucif0124 opened the pull request:
patch priority 불필요한 문구 제거,

심각도 정렬 문제 해결,

채팅, 사이드바 실시간 동기화 문제 해결했습니다.

lucif0124 committed:
위험도뱃지 통일

lucif0124 committed:
샘플분석페이지 수정(riskbadge)

lucif0124 committed:
Merge branch 'main' of https://github.com/s3zer0/system-security into feature/analysisSide

lucif0124 committed:
Merge branch 'main' of https://github.com/s3zer0/system-security into feature/analysisSide

lucif0124 committed:
Anaysis페이지 수정

lucif0124 committed:
lib탭 수정

lucif0124 committed:
patch 수정

s3zer0 commented:
[세트 #1] Docker 환경에서 외부로 노출된 API가 취약한 load() 함수를 사용하고 있어 즉각적인 패치가 필수입니다. CVSS 9.8의 CRITICAL 취약점으로 원격 코드 실행이 가능하며, 실제 악용 사례가 다수 보고되었습니다. 모든 작업을 중단하고 24시간 이내에 긴급 패치를 적용해야 합니다.

이건 여전히 발생하는 문제 같습니다.

s3zer0 commented:
프로세싱 처리까진 괜찮았는데, 프로세싱 진행 중 웹페이지를 새로고침하게되면 어떤게 분석중인지 안보이는 문제가 발생합니다. 이거도 확인 부탁드립니다.

lucif0124 committed:
Merge branch 'dev' of https://github.com/s3zer0/system-security into feature/analysisSide

lucif0124 committed:
patch 문구 수정

lucif0124 committed:
새로고침 문제 해결

s3zer0 committed:
Merge branch 'dev' into feature/analysisSide

lucif0124 committed:
Merge branch 'dev' of https://github.com/s3zer0/system-security into feature/analysisSide

lucif0124 committed:
Merge branch 'feature/analysisSide' of https://github.com/s3zer0/system-security into feature/analysisSide

lucif0124 committed:
UploadPanel 문제 해결

lucif0124 committed:
새로고침 오류 수정

lucif0124 commented:
patch 문구에 [#1]와 같은 불 필요한 문구 제거

도커파일 분석중일때 새로고침해도 분석하고있는 파일이 안보이는 현상 해결

lucif0124 committed:
폰트 수정, 샘플페이지 수정

lucif0124 committed:
불필요한 Mock-job 경로 수정

s3zer0 committed:
Merge branch 'dev' into feature/analysisSide

s3zer0 committed:
fix: little logic bugs.

  - 하이라이트 필터에서 심각도 문자열을 대문자로 통일해 비교하도록 바꿔서, “Critical/High”처럼 섞여 온 데이터도 제대로 잡아냅니다.
  - 존재하지 않는 cve 대신 백엔드가 주는 cve_id 필드를 쓰도록 고쳐, CVE 번호가 제대로 표시됩니다.
  - 이런 수정들은 코드와 샘플 데이터 기준으로 문제를 만들지 않고 오히려 안정성을 높입니다.
